velox/exec/tests/utils/TpchQueryBuilder.cpp

```diff
@@ -72,8 +72,10 @@ std::vector<std::string> mergeColumnNames(
 void TpchQueryBuilder::readFileSchema(
     const std::string& tableName,
     const std::string& filePath,
-    const std::vector<std::string>& columns) {
+    const std::vector<std::string>& columns,
+    const int32_t parquetPrefetchRowGroups) {
   dwio::common::ReaderOptions readerOptions{pool_.get()};
+  readerOptions.setPrefetchRowGroups(parquetPrefetchRowGroups);
```

Comment:
There is no use in plumbing `parquetPrefetchRowGroups` here as we are only reading the file metadata here.

Reply:
Thank you, following max_coalesce_bytes plumbing